recipes/grblas/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "grblas" %}
+{% set version = "1.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jim22k/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 91b05d38d8b245359ab2d8b89c066f40600ce82b5d7d2c680bf52e24e45d07d0
+
+build:
+  skip: true  # [win or py<37]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - cffi
+    - graphblas ==3.2.2
```

Does this need to be pinned so tightly?
```suggestion
    - graphblas
```

---

I'm not too familiar with `pin_subpackage` behavior. I'm using cffi to build a wrapper for the .h file in graphblas v3.2.2. Without the pin expression, will it still constrain it to `>=3.2,<3.3`?

---

It's always greater than the version used at build time. As it stands it pins only on the major version so the constraint will be `>=3.2.2,<4a0`. If that's not appropriate for the expected ABI stability it should be fixed in the `graphblas` recipe.

To pin for `>=3.2,<3.3a0` it would have to be:
```
  run_exports:
    - {{ pin_subpackage('graphblas', max_pin='x.x') }}
```

---

Sounds good. Thanks for looking at my recipe and helping me improve it.

---

No problem! 😄

As `run_exports` is applied at build time so it needs to be fixed and  have the new version deployed before this is merged. Shall I wait here and you can ping me when you've modified `graphblas`?

(Don't forget to bump the build number!)

---

Yeah, let me tighten the `graphblas` `pin_subpackage` specification, then ping you when it's merged and live.

---

@chrisburr : https://anaconda.org/conda-forge/graphblas/files shows the bumped version is now live, so we should be able to move forward with this PR.